src/services/TestStationsService.ts

```diff
@@ -27,7 +27,7 @@ class TestStationsService {
       Payload: toUint8Array(
         JSON.stringify({
           httpMethod: "GET",
-          path: `/test-stations/${testStationPNumber}`,
+          path: `/test-stations/${testStationPNumber}/email-addresses`,
```

Comment:
Will this endpoint ever return more than one email address?

Reply:
The rest of the code seems to suggest yes